evap/staff/templates/staff_semester_view.html

```diff
@@ -11,6 +11,7 @@ <h3>
         {% if request.user.is_manager %}
             &nbsp;
             <a href="{% url 'staff:semester_edit' semester.id %}" class="btn btn-sm btn-secondary">{% trans 'Edit' %}</a>
+            <button onClick="makeActiveSemesterModalShow({{ semester.id }})" class="btn btn-sm btn-danger" {% if semester.is_active %}disabled{% endif %} id="makeActiveSemesterButton" type="button">{% trans 'Make active' %}</button>
```

Comment:
please use `btn-light` instead of `btn-danger` and add an explanatory tooltip if the semester is currently active ("This is the active semester")

Reply:
Please remove the full stop, we usually don't have these in tooltips :)